types/block.go

```diff
@@ -723,3 +723,20 @@ func (blockID BlockID) Key() string {
 func (blockID BlockID) String() string {
 	return fmt.Sprintf(`%v:%v`, blockID.Hash, blockID.PartsHeader)
 }
+// Protobuf Compatiablity
+
+func (bc *Block) Unmarshal(bs []byte) error {
```

Comment:
These should be moved up to where the rest of the `Block` methods live

Reply:
@ebuchman, I have update the methods #2918, Sorry for inconvenience, I created new pull request due to changes occurred in Nodeinfo.go. Kindly request you verify it.